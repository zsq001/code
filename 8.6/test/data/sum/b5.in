100 100
1196393 1805339 1907719 1373003 3078358 1295718 1763287 1438991 28044 2873966 2651914 1242867 160561 2648344 913098 35700 2349445 3246438 2185569 2386287 905141 2774215 2322828 1678189 2038475 2506396 212533 1900584 3053432 81478 2359326 2006173 3182922 1230157 388704 2177581 26019 788851 2998821 2185494 1046776 1678539 791335 1666916 1210926 3233543 2557375 2783998 1964097 1477431 869646 1597589 2672323 1250455 2609323 990649 897434 699177 1972811 3262503 852443 1117632 1863759 745239 1528404 2096050 1030826 1862184 2667523 2292475 3049608 3304603 205469 1962575 977833 41756 3033452 1151388 337112 1185330 2856616 35262 1221124 1603986 1679750 1054054 2574434 1563911 2252068 217396 1348224 2272396 237315 3139016 982212 262748 1634502 916873 1556672 3130650 
1 17 69 36925
2 5 99
1 4 78 1746717
2 23 80
1 59 98 1123519
2 37 99
1 13 24 3261625
2 77 87
1 18 29 2749144
2 82 83
1 41 88 283609
2 27 76
1 6 71 1368783
2 1 98
1 62 92 2283186
2 25 99
1 64 69 3288023
2 30 63
1 12 72 727364
2 30 100
1 61 83 478530
2 6 34
1 85 86 3029743
2 30 81
1 93 100 2559746
2 81 98
1 20 36 2134540
2 50 95
1 69 84 3144432
2 42 86
1 49 73 2749116
2 71 98
1 5 5 3011026
2 2 58
1 21 59 1450663
2 34 83
1 28 58 938989
2 41 57
1 57 90 3050364
2 59 66
1 54 91 2809173
2 45 80
1 5 29 2748715
2 60 70
1 16 93 2108165
2 9 86
1 34 54 343283
2 54 59
1 71 82 2199566
2 34 58
1 27 47 2140277
2 59 72
1 43 93 1058517
2 37 93
1 20 37 1332033
2 27 48
1 79 92 1756850
2 44 84
1 54 100 1052576
2 40 49
1 48 63 2423532
2 28 30
1 11 83 1339666
2 33 83
1 37 61 2885132
2 24 28
1 27 80 1896022
2 6 15
1 29 70 1143115
2 68 98
1 1 45 497975
2 26 84
1 11 75 539867
2 12 26
1 18 47 2474676
2 17 90
1 52 98 836975
2 43 83
1 76 99 2494074
2 49 56
1 13 26 775312
2 45 77
1 85 99 2213680
2 4 95
1 27 38 1231843
2 31 74
1 8 17 2359287
2 24 63
1 5 32 1153941
2 57 76
1 27 100 640948
2 21 93
1 24 26 978154
2 32 63
1 1 11 2098944
2 47 47
1 11 13 2764290
2 39 79
1 85 87 1328826
2 14 24
