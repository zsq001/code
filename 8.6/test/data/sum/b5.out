6321379
103483912
660886138
416372981
860373725
982115329
155043701
200218419
91046022
182085068
22272374
791814952
105710102
51686782
416932428
21574317
789977001
552734285
120973913
101241451
939289992
1180800
837455964
945658572
573132464
587048108
710898984
978674533
564535209
544948602
948827812
884642974
636509068
926055431
303319937
822904271
486564595
152480678
386342989
29673514
203266940
469322125
664405439
545394142
348604133
471253829
5327597
438186282
34971411
830150185
